src/core/ext/transport/chttp2/transport/chttp2_transport.c

```diff
@@ -518,7 +520,7 @@ static void init_transport(grpc_exec_ctx *exec_ctx, grpc_chttp2_transport *t,
                   &channel_args->args[i], settings_map[j].integer_options);
               if (value >= 0) {
                 push_setting(exec_ctx, t, settings_map[j].setting_id,
-                             (uint32_t)value);
+                             (uint32_t)value, true);
```

Comment:
Good call! That write can actually catch all of the settings pushes from init_transport. I changed all the pushes in this function to false.